src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const KeyAndValues = sourceString.split(';');
```

Comment:
The variable name 'KeyAndValues' does not follow the convention of using camelCase for variable names. Consider renaming it to 'keyAndValues' to improve readability.